client/reports.go

```diff
@@ -42,6 +43,7 @@ func (c *Client) staticSendReport(gcas GCAServer, er EnergyRecord) {
 	eqr.Signature = glow.Sign(sb, c.staticPrivKey)
 	data := eqr.Serialize()
 	location := fmt.Sprintf("%v:%v", gcas.Location, gcas.UdpPort)
+	c.Log.Printf("udp send ShortID %v Timeslot %v PowerOutput %v", eqr.ShortID, eqr.Timeslot, eqr.PowerOutput)
```

Comment:
What's happening with these log messages? I want to make sure we don't have the possibility of writing a bunch of log messages to disk and tearing up the SD cards. Ideally these logs only go into memory and they can be printed to console if desired